lti_consumer/lti_1p3/exceptions.py

```diff
@@ -57,5 +57,13 @@ class LtiAdvantageServiceNotSetUp(Lti1p3Exception):
     pass
 
 
+class LtiNRPSServiceNotSetUp(Lti1p3Exception):
```

Comment:
Super-nit: elsewhere we use "Nrps" as the word in the Python name.  Other initialisms in this file are all Camelcase.

Reply:
Updated.